ldm/modules/attention.py

```diff
@@ -1,8 +1,11 @@
+from __future__ import annotations
```

Comment:
if this causes trouble for you you can just drop it; it's only included so I can specify the type hint `module: SelfAttention` on the `init_multihead` method.